ERCS/erc-X.md

```diff
@@ -0,0 +1,550 @@
+---
+eip: X
+title: Quantum Supremacy Puzzle
+author: Nicholas Papadopoulos (@nikojpapa)
+discussions-to: 
```

Comment:
Please create a discussions topic on Eth Magicians with a link to this PR

Reply:
Updated with the link (https://ethereum-magicians.org/t/erc-7826-quantum-supremacy-bounty/21866).